airflow/models.py

```diff
@@ -3022,6 +3024,21 @@ def xcom_pull(
             dag_id=dag_id,
             include_prior_dates=include_prior_dates)
 
+    def get_redirect_url(self, dttm, redirect_to):
```

Comment:
How about just `get_extra_links()`?

"redirect" calls to mind the server returning an HTTP 3xx, which this is not.

Reply:
Good point, can change this.